app/dashboard/gallery/orders/components/HistoryPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Don't forget to remove the react imports. It's not necessary. NextJS and React JS import react by default

Reply:
Done